config/certmanager/certificate_watcher.yaml

```diff
@@ -4,15 +4,27 @@
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
-  name: watcher-root
+  labels:
+    app.kubernetes.io/created-by: argo-cd
+    app.kubernetes.io/instance: kcp-lifecycle-manager
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: lifecycle-manager
+    app.kubernetes.io/part-of: kcp
+  name: klm-watcher-root
   namespace: istio-system
 spec:
   selfSigned: { }
 ---
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
-  name: watcher-serving
+  labels:
+    app.kubernetes.io/created-by: argo-cd
```

Comment:
this label should be defined in the `commonLabels`, which configured in the root Kustomization
https://github.com/kyma-project/lifecycle-manager/blob/2b4b9a82f9501b4d47fba0ef1f4a32ee3e1fc8dd/config/control-plane/kustomization.yaml#L5-L9